app/viewModels/P5/departure/DepartureStatusP5ViewModel.scala

```diff
@@ -309,4 +308,9 @@ object DepartureStatusP5ViewModel {
       DepartureStatusP5ViewModel("movement.status.P5.guaranteeWrittenOff", actions = Nil)
   }
 
+  private def errorsActionText(errors: Seq[String]): String = if (errors.length == 1) {
+    "viewError"
+  } else {
+    "viewErrors"
+  }
```

Comment:
This look fine however can we add some updated unit tests in the DepartureStatusP5ViewModelSpec please